dagshub/data_engine/model/datasource.py

```diff
@@ -529,6 +530,26 @@ def save_dataset(self, name: str):
         self.source.client.save_dataset(self, name)
         log_message(f"Dataset {name} saved")
 
+    def save_to_file(self, path: str = ".", name: str = ""):
```

Comment:
Could be useful to return the path to the file as a result of this function, so a Python script can do something else with the resulting file immediately (push it to dagshub, or w/e the user would want)

Reply:
Done